build-core-armv8m55-stm32n6.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Shell script for building Phoenix-RTOS firmware
+#
+# Builder for Phoenix-RTOS core components
+#
+# Copyright 2018-2024 Phoenix Systems
+# Author: Kaja Swat, Aleksander Kaminski, Pawel Pisarczyk
+#
+
+# fail immediately if any of the commands fails
+set -e
+
+b_log "Building phoenix-rtos-kernel"
+make -C "phoenix-rtos-kernel" all
+
+if [ "$LIBPHOENIX_DEVEL_MODE" = "y" ]; then
+	make -C "phoenix-rtos-kernel" install-headers
+
+	b_log "Building libphoenix"
+	make -C "libphoenix" all install
+fi
+
+b_log "Building libtty"
+make -C "phoenix-rtos-devices" libtty libtty-install
+
+b_log "Building libposixsrv"
+make -C "phoenix-rtos-posixsrv" libposixsrv libposixsrv-install
+
+b_log "Building phoenix-rtos-corelibs"
+make -C "phoenix-rtos-corelibs" all
```

Comment:
Adding separate comment, as answer to the AI comment might be not really visible - perhaps it is indeed good idea to `install` too